challenges/objects-arrays.js

```diff
@@ -14,6 +14,43 @@
 
 // Using your dinosaur objects, log answers to these questions:
 
+const tyrannosaurus = {
+  'object name': 'tyrannosaurus',
+  'diet': 'carnivorous',
+  'weight': '7000kg',
+  'length': '12m',
+  'period': 'Late Cretaceious',
+  'roar': function() {
```

Comment:
Declare roar outside of the tyrannosaurus object declaration.

Reply:
Check into this article on method definitions to make your method syntax shorter: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions

:)